Makefile

```diff
@@ -0,0 +1,87 @@
+.PHONY: deps
+deps: composer.json composer.lock
```

Comment:
why not depending on `vendor` directory and have a rule doing the `composer install`? Or is there a reason why you want to trigger a `composer install` multiple times?

Reply:
I need to change that,as well as making sure either composer can be ran inside of the containers, or run composer install in the e2e tests beforehand

Reply:
If you just want to run `composer install` every time, then you don't need to depend on anything. This should work just fine:
```
deps:
    composer install
```
I believe compose can quickly infer if there's a work to be done. So that's a non-issue.

Reply:
I meant for the e2e tests not the project itself

Reply:
Also see https://www.reddit.com/r/PHP/comments/7sltmh/makefile_for_lazy_developers/dt5rba0/.